### PR TITLE
add local-universe certificates to OS & dcos-binary certstore

### PR DESCRIPTION
**Pull request by juliusczar**
## Description
https://jira.mesosphere.com/browse/DCOS_OSS-1860

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures. :white_check_mark:
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, and 1.10). :white_check_mark:
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing). :white_check_mark:

### CODE REVIEW

**Comment by juliusczar** [comment]
sorry, havent seen it before (and cant access the jira task). this PR is partly a duplicate of: https://github.com/dcos/dcos-docs/pull/1259

File: 1.10/administering-clusters/deploying-a-local-dcos-universe.md [hunk 1]
```diff
@@ -199,14 +199,28 @@ You can install and run DC/OS services on a datacenter without internet access w
     dcos node ssh --master-proxy --mesos-id=<mesos-id>
     ```
 
-1.  Use the following commands to download a copy of the DC/OS certificate locally and set it as trusted.
+1.  Use the following commands to download a copy of the DC/OS Local Universe certificate locally and set it as trusted for the docker daemon
```
Comment (jgehrcke) [inline 1.10/administering-clusters/deploying-a-local-dcos-universe.md]: While we're here we should IMO say "DC/OS Local Universe CA certificate"

**Review by jgehrcke — APPROVED** [review]
I believe this is generally good to go. I have added one improvement suggestions which I'd appreciate to be taken care of. Also, I'd like @gpaul to have another look.

Things like
```
sudo curl -o /var/lib/dcos/pki/tls/certs/local-universe-ca.crt http://master.mesos:8082/certs/domain.crt
```
are generally insecure, but we might want to still accept this patch, and iterate on it later on.

Has this procedure been tested?

**Comment by juliusczar** [comment]
thanks for the feedback.
i dont cling on the curl way, i used this method analog to the existing example.
this procedurce is tested and runs in production - its necessary to get some of the offline universe frameworks to work (especially cassandra doesnt work without this fix).

File: 1.10/administering-clusters/deploying-a-local-dcos-universe.md (hunk 1 above)
Comment (juliusczar) [reply 1.10/administering-clusters/deploying-a-local-dcos-universe.md]: yeah, ill take care of it and update it tomorrow.

**Comment by juliusczar** [comment]
i close this PR because its kind of a doupe of the already merged https://github.com/dcos/dcos-docs/pull/1259 :)

**Pull request closed without merging**